jobs/openshift/cluster/integreatly/osd-cluster-integreatly-install.yaml

```diff
@@ -92,4 +95,19 @@
                 )
             }  
           }
+          
+          stage('Clone QE repository') {
+            dir('.') {
+              git branch: "master", url: "https://gitlab.cee.redhat.com/integreatly-qe/integreatly-qe.git"
+            }
+          }
+
+          stage('Update client secret in SSO') {
+            dir('test-suites/backend-testsuite') {
+              sh '''
+                npm install
+                node ssoClientUpdate.js ${clientSecret}
```

Comment:
I think there are some parameters required to run this script (SSO_URL, CLUSTER_URL and OPENSHIFT_TOKEN). I can't see them anywhere here. Am I missing something?

Reply:
Oh yes you're right. I totally forgot about them. The token is available...but not passed. And others (especially the SSO_URL) are missing. Thanks

Reply:
Well we'll see if it works...